crates/chainio/clients/avsregistry/src/reader.rs

```diff
@@ -399,6 +399,44 @@ impl AvsRegistryChainReader {
         Ok(quorum_stakes)
     }
 
+    /// Query registration detail
+    ///
+    /// # Arguments
+    ///
+    /// * `operator_address` - The operator address.
+    ///
+    /// # Returns
+    ///
+    /// A vector of booleans, where each boolean represents if the operator
+    /// is registered for a quorum.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if the operator id cannot be fetched or if the quorum bitmap
+    pub async fn query_registration_detail(
+        &self,
+        operator_address: Address,
+    ) -> Result<Vec<bool>, AvsRegistryError> {
+        let operator_id = self.get_operator_id(operator_address).await?;
+
+        let provider = get_provider(&self.provider);
+        let registry_coordinator =
+            RegistryCoordinator::new(self.registry_coordinator_addr, &provider);
+        let quorum_bitmap = registry_coordinator
+            .getCurrentQuorumBitmap(operator_id)
+            .call()
+            .await?;
+
+        let inner_value = quorum_bitmap._0.into_limbs()[0];
+
+        let mut quorums = Vec::<bool>::new();
+        for i in 0..64_u64 {
```

Comment:
Note that this implementation returns a vec of constant length 64. Instead, the go implementation returns an array that has the minimum required length to hold the registered quorums